### ISSUE TITLE
Support Jalali Calander & persian Numbering

### ISSUE THREAD

**Issue by musapha** [eternicode/bootstrap-datepicker#1780]
Dear all , first i would like to ask for the Great job you have done and appreciate your contribution . 
as the bootstrap-picketer is way professional and easy to use , if you could help me with Jalali calendar  (Persian calendar ) which is the default calendar for Persians , Kazakhstan , Tajikistan , and Afghanistan   and all Farsi speaking people . i would like you to help me with a way to change the calendar to jalali calendar and Persian numerals format. which would be awesome , 
if you could even help me how could i add these features and where to start that could be something as well . 
again i do appreciate your help in advance . thanks .

**Comment by musapha** [comment]
Since this datepicker is so commonly use in may open source software , this is also a great boot for many people all over the world .

**Comment by reza239** [comment]
Yeah, I also need to convert it to **Jalali Calendar**. I think it's the best datepicker.
Please find a solution to improve it. Thanks.

**Comment by reza239** [comment]
How can I add 31 days in first 6 month of calendar?
Because in Jalali 0-5 month have 31 days.

**Comment by bahmansh** [comment]
Hi
Im ready for helping for doing this.

How can i help you dear?

**Comment by bndghn** [comment]
please  add Persian date to this calendar.
tanks.

**Comment by jafaripur** [comment]
This project use Jalali as calendar with changing locale:

https://github.com/fullcalendar/fullcalendar
https://fullcalendar.io/demos
